.flake8

```diff
@@ -0,0 +1,6 @@
+[flake8]
+max-line-length = 99
+exclude =
+    __pycache__
+    tests
```

Comment:
We should probably include the pytest code in the flake8 checks.

Reply:
Good point.